lms/djangoapps/course_api/blocks/transformers/milestones.py

```diff
@@ -66,8 +66,6 @@ def user_gated_from_block(block_key):
 
             if usage_info.has_staff_access:
                 return False
-            elif self.has_pending_milestones_for_user(block_key, usage_info):
```

Comment:
@BessieSteinberg @bill-filler 
1. Search the codebase for `include_special_exams`.
2. You'll want to update all 5 of those places (except for the 6th one in the code in this file) to now also use `include_gated_sections`.
3. Update this line of code to check for `self.include_gated_sections`.

Reply:
I forgot to mention: 
  4. Also do a code search on `special_exam_info`. You only need to be concerned with the following 2 code places.
  5. Update the [call that the browser code](https://github.com/edx/edx-platform/blob/7f369b1ed951823e4b753d3e0a4598ed28b1d63c/openedx/features/course_experience/utils.py#L70) makes to include the new requested_field (`gated_sections`).  This way, only the browser's path is updated and not the mobile path.
  6. (This should have also come up in the search on step 1) Update the [get_blocks](https://github.com/edx/edx-platform/blob/7f369b1ed951823e4b753d3e0a4598ed28b1d63c/lms/djangoapps/course_api/blocks/api.py#L58) python code to check for the `gated_sections` requested_field and pass it into `MilestonesAndSpecialExamsTransformer`.